nengo_loihi/tests/test_loihi_api.py

```diff
@@ -22,3 +25,71 @@ def test_overflow_signed(b):
     y, o = overflow_signed(x, bits=b)
     assert np.array_equal(y, z)
     assert np.array_equal(o, q)
+
+
+@pytest.mark.parametrize('offset', (0, 1))
+def test_decay_magnitude(offset, plt, logger):
+    bits = 12
+    # decays = np.arange(1, 2**bits)
+    decays = np.arange(1, 2**bits, 7)
+    ref = []
+    emp = []
+    est = []
+
+    for decay in decays:
+        def empirical_decay_magnitude(decay, x0, invert=False):
```

Comment:
The `invert` arg is never used, and it feels like a weird thing to make part of the function when you could just do `1.0 / empirical_decay_magnitude(...)` instead. I'd remove it.